codec-http/src/main/java/io/netty5/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -210,9 +211,9 @@ public long forceCloseTimeoutMillis() {
 
     /**
      * Flag to indicate if the closing handshake was initiated because of timeout.
-     * For testing only.
```

Comment:
Let's keep the comment.

Reply:
This of course applies to many other places in the PR. I decided to stop pointing them out after this one.

Reply:
Got it. for sure. Let me revert changes on the comments.

Reply:
I've just changed it. May I request your review?